agent/php_execute.c

```diff
@@ -640,7 +640,7 @@ static void nr_php_show_exec(NR_EXECUTE_PROTO TSRMLS_DC) {
     || defined OVERWRITE_ZEND_EXECUTE_DATA
         nr_php_op_array_get_wraprec(NR_OP_ARRAY TSRMLS_CC) ? " *" : "",
 #else
-        nr_php_get_wraprec_by_name(execute_data->func) ? " *" : "",
+        nr_php_get_wraprec_by_func(execute_data->func) ? " *" : "",
```

Comment:
Is it generally safe to assume `execute_data` is not NULL?

Reply:
This just got formatted here and isn't related to this PR.
I think this comment should be directed to the https://github.com/newrelic/newrelic-php-agent/pull/555 ticket which is modifying the function.
Doesn't hurt to check execute_data and func for NULL.  NR_OP_ARRAY  also uses execut_data.

Reply:
The old instrumentation didn't have any NULL checks for `execute_data` received from Zend engine: no NULL checks for `execute_data` when agent receives control in `nr_php_execute ` ([agent/php_execute.c:1336](https://github.com/newrelic/newrelic-php-agent/blob/main/agent/php_execute.c#L1336)); no NULL checks [here](https://github.com/newrelic/newrelic-php-agent/blob/main/agent/php_execute.c#L1287) or [here](https://github.com/newrelic/newrelic-php-agent/blob/main/agent/php_execute.c#L1126). I think the reason for this approach might have been that if the agent receives NULL `execute_data`, then something is fundamentally wrong within the Zend engine and maybe it's better to crash.